src/Symfony/Component/Notifier/Bridge/Slack/SlackTransportFactory.php

```diff
@@ -28,17 +29,20 @@ final class SlackTransportFactory extends AbstractTransportFactory
      */
     public function create(Dsn $dsn): TransportInterface
     {
-        $scheme = $dsn->getScheme();
+        if ('slack' !== $dsn->getScheme()) {
+            throw new UnsupportedSchemeException($dsn, 'slack', $this->getSupportedSchemes());
+        }
+
+        if ('/' !== $dsn->getPath()) {
```

Comment:
The check looks fragile to me. I would check for the whole pattern instead via a regex.

Reply:
I have thought for a regex but i couldn't think of any which makes sense for me, sorry :(

At the end i think it's just not allowed to add a path to the DSN. It wouldn't be processed anyway.